test/API/driver/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 2.8.12.2 FATAL_ERROR)
+project(H5_API_TEST_DRIVER CXX)
```

Comment:
This has a language requirement for C++ -  another guard for the root/CMakeLists.txt option.